### PR TITLE
Javascript Lint pipeline

### PR DESCRIPTION
**Pull request by mateiw**
JavaScript pipeline to run ESLint and Flow on PRs.

### CODE REVIEW

File: jenkins_pipelines/manager_prs/Jenkinsfile_prs_javascript_lint [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    jslint_test = "gitarro.ruby2.1  -r SUSE/spacewalk" + 
+                " -c javascript_lint -d \"javascript-lint\" " +
+                " -f \"js/\" " +
+                " -t susemanager-utils/testing/automation/javascript-lint.sh" +
+                " -u ${env.BUILD_URL} " +
+                " -g ${env.WORKSPACE} " 
+   }
+   // run only on specific hosts
+    agent { label 'suse-manager-unit-tests' }
+   // trigger
```
Comment (MalloZup) [inline jenkins_pipelines/manager_prs/Jenkinsfile_prs_javascript_lint]: how do we schedule this pipeline if not auto-triggered?

File: jenkins_pipelines/manager_prs/Jenkinsfile_prs_javascript_lint [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  // env variables
+  environment { 
+
+    jslint_check = "gitarro.ruby2.1  -r SUSE/spacewalk" + 
+                " -c javascript_lint -d \"javascript-lint\" " +
+                " -f \".js\" " +
+                " -t susemanager-utils/testing/automation/javascript-lint.sh" +
+                " -g ${env.WORKSPACE} " +
+                "--check " // --changed_since 3600
```
Comment (MalloZup) [inline jenkins_pipelines/manager_prs/Jenkinsfile_prs_javascript_lint]: if we comment out this we will reach the Rate limit of github ( `changed_since`)

File: jenkins_pipelines/manager_prs/Jenkinsfile_prs_javascript_lint [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+pipeline {
+
+  // env variables
+  environment { 
+
+    jslint_check = "gitarro.ruby2.1  -r SUSE/spacewalk" + 
+                " -c javascript_lint -d \"javascript-lint\" " +
+                " -f \".js\" " +
+                " -t susemanager-utils/testing/automation/javascript-lint.sh" +
```
Comment (MalloZup) [inline jenkins_pipelines/manager_prs/Jenkinsfile_prs_javascript_lint]: `nitpick`: we can have this script inside of the pipeline, But is also ok in manager

**Comment by mateiw** [comment]
@MalloZup I addressed your comments apart from `javascript-lint.sh`. I would keep it inside manager for consistency with the other scripts like `java-checkstyle.sh`, etc.

**Pull request merged**